op-batcher/metrics/metrics.go

```diff
@@ -109,6 +114,16 @@ func NewMetrics(procName string) *Metrics {
 			Name:      "pending_blocks_count",
 			Help:      "Number of pending blocks, not added to a channel yet.",
 		}, []string{"stage"}),
+		pendingBlocksBytesCount: factory.NewCounter(prometheus.CounterOpts{
+			Namespace: ns,
+			Name:      "pending_blocks_bytes_count",
+			Help:      "Size of transactions in pending blocks as they are fetched from L2",
+		}),
+		pendingBlocksBytesCurrent: factory.NewGauge(prometheus.GaugeOpts{
+			Namespace: ns,
+			Name:      "pending_blocks_bytes_gauge",
+			Help:      "Current size of transactions in the pending (fetched from L2 but not in a channel) stage.",
+		}),
```

Comment:
```suggestion
		pendingBlocksBytesTotal: factory.NewCounter(prometheus.CounterOpts{
			Namespace: ns,
			Name:      "pending_blocks_bytes_total",
			Help:      "Total size of transactions in pending blocks as they are fetched from L2",
		}),
		pendingBlocksBytesCurrent: factory.NewGauge(prometheus.GaugeOpts{
			Namespace: ns,
			Name:      "pending_blocks_bytes_current",
			Help:      "Current size of transactions in the pending (fetched from L2 but not in a channel) stage.",
		}),
```
I suggest we keep the naming of the metrics and variables more consistent with the other metrics. I like the idea of appending `current` to clearly differentiate from the `total` though. Less confusing.

Note that the [prometheus docs](https://prometheus.io/docs/practices/naming/) suggest that the metric names of counters end in `total`.
> Note that an accumulating count has `total` as a suffix, in addition to the unit if applicable.